client/components/course/Sidebar/Discussion/Comment.vue

```diff
@@ -7,6 +7,7 @@
         :size="38"
         :username="comment.author.email"
         :initials="authorInitials"
+        :src="user.imgUrl"
```

Comment:
Same question? What's is the case if there is no `imgUrl`?

Reply:
[Gravatar] (https://en.gravatar.com/site/implement/) covers that case on backend with`beforeCreate` and `beforeUpdate` user hooks. They always set gravatar image as default imgUrl value. Furthermore, if there is not any image on gravatar account, or even user mail doesn't match gravatars email, still gravatar will provide default image (in my case 'mystery person' image).

Reply:
Noice.